Makefile

```diff
@@ -474,6 +484,57 @@ contribs/gmf/build/mobile.css: $(GMF_APPS_MOBILE_LESS_FILES) \
 	mkdir -p $(dir $@)
 	./node_modules/.bin/lessc contribs/gmf/apps/mobile/less/build.less $@ --autoprefix
 
+
+# i18n
+
+# if don't exists create one for read only access
+$(HOME)/.transifexrc:
+	echo "[https://www.transifex.com]" > $@
+	echo "hostname = https://www.transifex.com" >> $@
+	echo "username = c2c" >> $@
+	echo "password = c2cc2c" >> $@
+	echo "token =" >> $@
+
+.tx/config: .tx/config.mako .build/python-venv/bin/mako-render
+	PYTHONIOENCODING=UTF-8 .build/python-venv/bin/mako-render \
+		--var "git_branch=$(TX_GIT_BRANCH)" $< > $@
+
+#.build/locale/ngeo.pot: lingua.cfg .build/node_modules.timestamp \
+#		$(NGEO_DIRECTIVES_PARTIALS_FILES) $(NGEO_JS_FILES)
+#	mkdir -p $(dir $@)
+#	node buildtools/extract-messages.js $(NGEO_DIRECTIVES_PARTIALS_FILES) $(NGEO_JS_FILES) > $@
+
+.build/locale/gmf.pot: lingua.cfg .build/node_modules.timestamp \
+		$(GMF_DIRECTIVES_PARTIALS_FILES) $(GMF_JS_FILES)
+	mkdir -p $(dir $@)
+	node buildtools/extract-messages $(GMF_DIRECTIVES_PARTIALS_FILES) $(GMF_JS_FILES) > $@
+
+.build/python-venv/bin/tx: .build/python-venv $(HOME)/.transifexrc
+	.build/python-venv/bin/pip install transifex-client
+	touch $@
+
+.PHONY: transifex-send
+transifex-send: .tx/config .build/python-venv/bin/tx \
+		.build/locale/gmf.pot
+		# .build/locale/ngeo.pot
+	.build/python-venv/bin/tx push --source
+
+#.build/locale/%/LC_MESSAGES/ngeo.po: .tx/config .build/python-venv/bin/tx
```

Comment:
I add those blocks to be almost ready when we will have something to translate in ngeo ...